### ISSUE TITLE
collect2 error

### ISSUE THREAD

**Issue by jorinryms** [rpav/xleapmouse#6]
Most of the build process works correctly, except for:

/usr/bin/ld: cannot find -lLeap
collect2: error: ld returned 1 exit status
make[2]: *** [src/CMakeFiles/xleapmouse.dir/build.make:129: src/xleapmouse] Error 1
make[2]: Leaving directory '/mnt/NFS/Programs/LeapDeveloperKit_2.3.1+31549_linux/xleapmouse'
make[1]: *** [CMakeFiles/Makefile2:94: src/CMakeFiles/xleapmouse.dir/all] Error 2
make[1]: Leaving directory '/mnt/NFS/Programs/LeapDeveloperKit_2.3.1+31549_linux/xleapmouse'
make: *** [Makefile:130: all] Error 2

I don't know if it's a problem with build.make or if I failed to link something properly when installing the SDK.

**Comment by rpav** [comment]
Yeah, looks like missing sdk stuff; note the first line with missing `-lLeap`.

**Comment by rpav** [comment]
... the solution here being, verify the packages installed the right things (`libLeap.so...`) and in a place they're normally found.  You can point cmake at them with custom linker flags in a pinch of course.
